### DISCUSSION TITLE
Game stopped responding to input but continued playing as normal

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#22245] [Q&A]
### Type

Other

### Bug description

In the middle of playing a beatmap, the game suddenly stopped responding to input, including attempts to quit. During this time, the system cursor displayed the "beachball". The beatmap continued playing as normal. Upon force quitting, an os prompt appeared saying that osu quit unexpectedly (crash dump linked below). [This is the map I was playing when the issue occurred](https://lazer.ppy.sh/beatmapsets/925261#taiko/3639165)

### Screenshots or videos

The cursor is 'beachballing' and I am mashing keys during this entire clip
https://user-images.githubusercontent.com/100527514/212764390-28d84299-1d13-4c11-9716-96cde42fd807.mov

### Version

2022.1228.0-lazer

### Logs

[Mac crash dump](https://docs.google.com/document/d/1pEYgso9zT7wdUS-51XI-hU93h4YJY44y1MH_2UGDNkQ/edit?usp=sharing)
[database.log](https://github.com/ppy/osu/files/10429441/database.log)
[input.log](https://github.com/ppy/osu/files/10429442/input.log)
[network.log](https://github.com/ppy/osu/files/10429443/network.log)
[performance-draw.log](https://github.com/ppy/osu/files/10429444/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/10429445/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/10429446/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10429447/runtime.log)

**Comment by bdach** [comment]
Downgrading to discussion for now unless specific repeatable reproduction steps are provided.
